packages/firebase_database/firebase_database_web/lib/firebase_database_web.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright 2017, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library firebase_database_web;
+
+import 'dart:async';
+
+import "package:firebase/firebase.dart" as firebase;
+import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_database_platform_interface/firebase_database_platform_interface.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+
+part './src/database_reference_web.dart';
+part './src/ondisconnect_web.dart';
+part './src/query_web.dart';
+part './src/utils/snapshot_utils.dart';
+
+/// Web implementation for [DatabasePlatform]
+/// delegates calls to firebase web plugin
+class FirebaseDatabaseWeb extends DatabasePlatform {
+  /// Instance of Database from web plugin
+  final firebase.Database _firebaseDatabase;
+
+  /// Called by PluginRegistry to register this plugin for Flutter Web
+  static void registerWith(Registrar registrar) {
+    DatabasePlatform.instance = FirebaseDatabaseWeb();
+  }
+
+  /// Builds an instance of [DatabaseWeb] with an optional [FirebaseApp] instance
+  /// If [app] is null then the created instance will use the default [FirebaseApp]
+  FirebaseDatabaseWeb({FirebaseApp? app, String? databaseURL})
+      : _firebaseDatabase = firebase.database(firebase.app(app?.name)),
```

Comment:
Here how can we connect to different databases of the app here?
How can we get the database instance from the database URL here?

According to the web documentation, we can do that with the database URL.
https://firebase.google.com/docs/reference/js/database.md#getdatabase

Reply:
`firebase.app().database(url)` see the v8 JS SDK reference docs; https://firebase.google.com/docs/reference/js/v8/firebase.app.App#database

Reply:
@deepak786 let's use our own JS interop instead of relying on firebase-dart package directly. Have a look at `cloud_firestore_web/lib/src/interop` for a reference

Reply:
> @deepak786 let's use our own JS interop instead of relying on firebase-dart package directly. Have a look at `cloud_firestore_web/lib/src/interop` for a reference

For this we can copy paste existing `firebase-dart` code over for `database` as a starting point into firebase_database_web/lib/src/interop` - then we can make the changes we need to support this in our own codebase going forwards - since this will allow us to move quicker for this and all future changes

Reply:
I have added the interop (copied from firebase-dart and modified).
Created the function that will return the database instance with databaseURL https://github.com/FirebaseExtended/flutterfire/blob/41b73e82acfd2b8340de327d02eb21ed3ecfb532/packages/firebase_database/firebase_database_web/lib/src/interop/database.dart#L23